### PR TITLE
moving denominations under common

### PR DESCRIPTION
**Pull request by alajko**
## Issue

removing ether and moving denominations under common.

### CODE REVIEW

File: common/denominations/denominations.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package denominations
+
+//ONE Units
+const (
+	Wei  = 1
```
Comment (harmony-ek) [inline common/denominations/denominations.go]: `Atto`?

File: common/denominations/denominations.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	GWei = 1e9
```
Comment (harmony-ek) [inline common/denominations/denominations.go]: `Nano`?

Comment (alajko) [reply common/denominations/denominations.go]: done

File: common/denominations/denominations.go (hunk 1 above)
Comment (alajko) [reply common/denominations/denominations.go]: done

**Pull request merged**